content/en/infrastructure/process/_index.md

```diff
@@ -144,7 +134,7 @@ For example:
     "containerDefinitions": [
         {
             "name": "datadog-agent",
-            "image": "public.ecr.aws/datadog/agent:7.50.0-rc.1",
+            "image": "public.ecr.aws/datadog/agent",
```

Comment:
Let's use the same as [here](https://docs.datadoghq.com/integrations/ecs_fargate/?tab=cloudformation) - with `:latest` explicitly set:
```suggestion
            "image": "public.ecr.aws/datadog/agent:latest",
```